### DISCUSSION TITLE
I think the portable settings folders for Chromium/Chrome browsers or Electron apps be moved to the `Appdata\Roaming` folder.

### DISCUSSION THREAD

**Discussion by kris3713** [ScoopInstaller/Extras#10651] [General]
Apps that are not meant to be portable should not be forced to be portable. Notice how all Chromium/Chrome browsers or Electron apps (except VSCode & VSCodium) always create 2 instances. Even if you use a parameter like `--user-data-dir="$dir\User Data"`, it will still another config in the `Appdata\Roaming` folder that is out of Scoop's control. Here is my suggestion, change all manifests that contain Chromium/Chrome browsers or Electron apps, and add a script that will copy the portable settings folders into the `Appdata\Roaming` folder. This should resolve the bug of Chromium/Chrome browsers or Electron apps creating 2 instances.

Do you think these changes should be made, or should we just let these bugs with those apps slide under the bus?

**Comment by HUMORCE** [comment]
I agree with this, which is why I just use Scoop to install firefox and ungoogled-chromium, while Google Chrome uses the official installer.
